Abot2.Tests.Unit/Core/HyperlinkParserTest.cs

```diff
@@ -282,7 +282,7 @@ public void GetLinks_RelativeBaseTagPresent_ReturnsRelativeLinksPageUri()
         [TestMethod]
         public void GetLinks_InvalidBaseTagPresent_ReturnsRelativeLinksPageUri()
         {
-            _crawledPage.Content.Text = "<base href=\"http:http://http:\"><a href=\"http://aaa.com/\" ></a><a href=\"/aaa/a.html\" /></a>";
+            _crawledPage.Content.Text = "<base href=\"////images/\"><a href=\"http://aaa.com/\"></a><a href=\"/aaa/a.html\"></a><a href=\"aaa/a.html\"></a>";
```

Comment:
Technically, the value of the base tag didn't have to be updated here, if the goal simply is to have a passing test. Leaving the base tag value as it was, does however mean that it's no longer testing what it was previously - which is, "what happens if an URI instance can't be created from it".